File: W1/square1.c
#include <stdio.h>

int main() {
    int size;
    
    scanf("%d", &size);
    
    for ( int column = 1; column <= size; column++ ) {
        for ( int row = 1; row < size; row++ ) {
            printf("%d ", row);
        }
        printf("%d\n", size);
    }
    
    return 0;
}

File: PHP/square1.php
<?php
    $size = fgets(STDIN);
    
    for ( $column = 1; $column <= $size; $column++ ) {
        for ( $row = 1; $row < $size; $row++ ) {
            echo $row . ' ';
        }
        // echo $size . PHP_EOL; # for apple, linux 
        echo $size; # for windows
    }
?>
